src/analysis/special_functions/gamma/beta.lean

```diff
@@ -507,3 +508,48 @@ end
 end real
 
 end gamma_reflection
+
+section inv_gamma
+open_locale real
+
+namespace complex
+/-! ## The reciprocal Gamma function
+
+We show that the reciprocal Gamma function `1 / Γ(s)` is entire. These lemmas show that (in this
+case at least) mathlib's conventions for division by zero do actually give a mathematically useful
+answer! (These results are useful in the theory of zeta and L-functions.) -/
+
+/-- A reformulation of the Gamma recurrence relation which is true for `s = 0` as well. -/
+lemma one_div_Gamma_eq_self_mul_one_div_Gamma_add_one (s : ℂ) :
+  1 / Gamma s = s * (1 / Gamma (s + 1)) :=
+begin
+  rcases ne_or_eq s 0 with h | rfl,
+  { rw [Gamma_add_one s h, mul_one_div, ←div_div, div_self h] },
+  { rw [zero_add, Gamma_zero, div_zero, zero_mul] }
+end
+
+/-- The reciprocal of the Gamma function is differentiable everywhere (including the points where
+Gamma itself is not). -/
+lemma differentiable_one_div_Gamma : differentiable ℂ (λ s : ℂ, 1 / Gamma s) :=
+begin
+  suffices : ∀ (n : ℕ), ∀ (s : ℂ) (hs : -s.re < n), differentiable_at ℂ (λ u : ℂ, 1 / Gamma u) s,
+    from λ s, let ⟨n, h⟩ := exists_nat_gt (-s.re) in this n s h,
```

Comment:
```suggestion
   { from λ s, let ⟨n, h⟩ := exists_nat_gt (-s.re) in this n s h },
```

Reply:
I thought the whole point of `from` as a synonym for `exact` was to allow the style `have / suffices x, from some_term` without the braces?

Reply:
Yes, I would say that the braces are needed to delineate subproofs. But a `from` or `by` also serves that purpose.